byzcoin/struct.go

```diff
@@ -674,24 +674,24 @@ func (c ChainConfig) checkNewRoster(newRoster onet.Roster) error {
 // Here is an example of what it outputs:
 //
 // ```
-// ChainConfig
-// - BlockInterval: 7s
-// - Roster: {1e89775c-636a-536a-bc39-1ec951c86dc9 [tls://localhost:2002 tls://localhost:2004 tls://localhost:2006] 467abd382f78222e898d323194c0fb30f7096bb6bb885ea31284979f794e558a}
-// - MaxBlockSize: 5000000
-// - DarcContractIDs:
-// -- darc contract ID 0: darc
-// -- darc contract ID 1: darc2
-// -- darc contract ID 2: darc3'
+// - ChainConfig:
+// -- BlockInterval: 7s
+// -- Roster: {1e89775c-636a-536a-bc39-1ec951c86dc9 [tls://localhost:2002 tls://localhost:2004 tls://localhost:2006] 467abd382f78222e898d323194c0fb30f7096bb6bb885ea31284979f794e558a}
+// -- MaxBlockSize: 5000000
+// -- DarcContractIDs:
+// --- darc contract ID 0: darc
+// --- darc contract ID 1: darc2
+// --- darc contract ID 2: darc3'
 // ```
 func (c ChainConfig) String() string {
 	var res strings.Builder
```

Comment:
```suggestion
	res := new(strings.Builder)
```

And then all the other references to res do not need an &.